project/Build.scala

```diff
@@ -805,6 +805,74 @@ object Build {
       javaOptions := (javaOptions in `scala3-compiler-bootstrapped`).value
     )
 
+  /** Scala library compiled by dotty using the latest published sources of the library */
+  lazy val `stdlib-bootstrappedject.in(file("stdlib-stdlib-bootstrapped
+    withCommonSettings(Bootstrapped).
+    dependsOn(dottyCompiler(Bootstrapped) % "provided; compile->runtime; test->test").
+    dependsOn(`scala3-tasty-inspector` % "test->test").
+    settings(commonBootstrappedSettings).
+    settings(
```

Comment:
You could add a setting:
```suggestion
    settings(
      moduleName := "scala-library"
```
this way the jar will be called scala-library_3.0.0-M1.jar which makes sense.

Reply:
Added